keg/tests/test_db.py

```diff
@@ -7,6 +7,7 @@
 from keg.signals import db_init_pre, db_init_post, db_clear_post, db_clear_pre
 from keg.testing import invoke_command
 
+#from keg_apps.db.blog.model.entities import Blog
```

Comment:
Is this accidental?

Reply:
At first, I meant to remove the line entirely, but decided it needed to remain commented. What I forgot was a comment as to why Blog should not be imported there.